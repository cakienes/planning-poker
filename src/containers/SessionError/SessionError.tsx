import React from 'react';
import Layout from '../Layout/Layout';

const SessionError: React.FC<any> = () => {
    return (
        <Layout>
            <div>SESSION IS NOT FOUND</div>
        </Layout>
    );
};

export default SessionError;
