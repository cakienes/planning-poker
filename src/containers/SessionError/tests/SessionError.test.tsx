import React from 'react';
import { createRenderer, ShallowRenderer } from 'react-test-renderer/shallow';
import SessionError from '../SessionError';

describe('SessionError.tsx', () => {
    let renderer: ShallowRenderer;

    beforeEach(() => {
        renderer = createRenderer();
    });

    it('renders correctly SessionError', async () => {
        expect(renderer.render(<SessionError />)).toMatchSnapshot();
    });
});
