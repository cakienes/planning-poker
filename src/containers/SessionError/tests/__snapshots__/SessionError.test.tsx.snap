// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SessionError.tsx renders correctly SessionError 1`] = `
<Layout>
  <div>
    SESSION IS NOT FOUND
  </div>
</Layout>
`;
